src/utils/exif/index.ts

```diff
@@ -0,0 +1,25 @@
+export function prettyPrintExif(data: Object) {
+    let a;
```

Comment:
What's `a` here? Wondering if it's possible to rename it to something more readable?

Reply:
yes..I shoud have done that...Just copied and pasted the code from a library 😬

Reply:
will update the code